voice-output/voice-pack-voicer.js

```diff
@@ -23,9 +23,35 @@ function weightedRandom(weights) {
 }
 
 class VoicePack {
-  constructor(syllableFiles, audioBuffer) {
+  constructor(syllableFiles, actionFiles, voiceActor, audioBuffer) {
     this.syllableFiles = syllableFiles;
+    this.actionFiles = actionFiles;
     this.audioBuffer = audioBuffer;
+    this.voiceActor = voiceActor;
+    this.actionVoices = actionFiles.map(({name, offset, duration}) => {
```

Comment:
The entire voicepack is one set of files right? If true then we should only pass one set of files here.

Maybe `syllableFiles` should just be `files` and has a bad name?

Reply:
change `syllableFiles` to `files`